contracts/mesh-provider/src/contract.rs

```diff
@@ -115,17 +120,7 @@ pub fn execute_receive_claim(
         return Err(ContractError::ZeroAmount);
     }
 
-    let mut val = VALIDATORS
```

Comment:
We actually probably want this check here, even if we don't use the result.

Erroring quickly when they try to bond to random address that doesn't represent a validator, rather than handling that in `fail_stake` is much better UX.

Reply:
Good point, will add it back.